app/client/views/application/applicationCtrl.js

```diff
@@ -133,7 +133,15 @@ angular.module('reg')
         });
       }
 
+      $scope.activateCharCount = false
 
+      $scope.$watch(
+        "user.profile.essay.length",
+        function (newValue, oldValue, scope){
+          if ($scope.activateCharCount === false & newValue > 100)
```

Comment:
`$scope.activateCharCount === false` can be shortened to `!$scope.activateCharCount`. also `&` is a bitwise operator